datastore/integration-test/response/collection/area/200/data_within_an_area_with_two_parameters.json

```diff
@@ -52,35 +52,43 @@
                 ]
             },
             "parameters": {
-                "relative_humidity_2.0_mean_PT1M": {
+                "wind_speed_10_mean_PT10M": {
```

Comment:
What is the order that is used? As this is not alphabetical. Is it fixed? Collection endpoints does seem to be ordered? Should really be the same.